app/serializers/articles_index_serializer.rb

```diff
@@ -16,4 +16,8 @@ def image
   def rating
     Rating.where(article_id: object.id).average(:rating).to_f
   end
+
+  def author
+    "#{object.user['first_name']} #{object.user['last_name']}" 
+  end
```

Comment:
Would be nice to have it in the same format as in the old show action, so we don't have to change it on the clients :))

Reply:
Fixed.